docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -81,6 +81,56 @@ Each Protocol simulation Unit carries out the following steps:
 
 Note: three different types of multistate sampling (i.e. replica swapping between lambda states) methods can be chosen; HREX, SAMS, and independent (no lambda swaps attempted). By default the HREX approach is selected, this can be altered using ``simulation_settings.sampler_method`` (default: ``repex``).
 
+Analysis
+~~~~~~~~
+
+As standard, some analysis of the each simulation repeat is performed.
+This analysis is made available through either the dictionary of results in the execution output,
+or through some ready-made plots for quick inspection.
+This analysis can be categorised as relating
+to the energetics of the different lambda states that were sampled,
+or to the analysis of the change in structural conformation over time in each state.
+
+.. todo: issue 792, consolidate this page into its own analysis page and link both RBFE and AFE pages to it
+
+* Energetic and replica exchange analysis.  These analyses consider the swapping and energetic overlap between the
+  different simulated states to help assess the convergence and correctness of the estimate of free energy
+  difference produced.
+
+  * MBAR overlap matrix.  This plot is used to assess if the different lambda states simulated overlapped energetically.
+    Each matrix element represents the probability of a sample from a given row state being observable in a given column
+    state.
+    This plot should show that the diagonal of the matrix has some "width" so that the two end states are connected,
+    with off-diagonal elements being at least 0.03[2]_.
+  * Replica exchange overlap matrix (for repex type simulations only).  Similar to the MBAR overlap matrix, this shows
```

Comment:
```suggestion
  * Replica exchange probability matrix (for repex type simulations only).  Similar to the MBAR overlap matrix, this shows
```
The original API docs are incorrect here, it's not an overlap matrix, it's a probability of state exchange.